crates/iceberg-catalog/src/implementations/postgres/catalog.rs

```diff
@@ -576,6 +576,7 @@ impl Catalog for super::PostgresCatalog {
             &catalog_state.read_pool(),
             None,
             pagination_query,
+            false,
```

Comment:
If we hook into the proc macro of migration, we could remove it now.
This would enable users to skip 0.5 on migrations